src/flows/pipes/Visualization/ErrorThresholds/FieldColumnDropdown.tsx

```diff
@@ -37,8 +37,9 @@ const FieldColumnDropdown: FC<Props> = ({threshold, index}) => {
         return c.name === '_value'
       })
 
-      const fields = (results.parsed.table.columns['_field']
-        .data as any).reduce((acc, curr, index) => {
+      const fields = (
+        (results.parsed.table.columns['_field'].data as any) ?? []
```

Comment:
shouldn't it be an Array<any>?

Reply:
it should, but a base arrays without a type don't make TS happy, and declaring `string[]` would be a lie because it could be undefined. At this point `any` suffices